src/dex-ui/services/HederaService/GovernorService.ts

```diff
@@ -165,6 +169,44 @@ const sendCreateTransferTokenProposalTransaction = async (
   return proposalTransactionResponse;
 };
 
+interface CreateContratctUpgradeProposalParams {
+  title: string;
+  description: string;
+  linkToDiscussion: string;
+  contarctId: string;
+  proxyId: string;
+  signer: HashConnectSigner;
+}
+
+/**
+ * TODO
+ * @param params -
+ * @returns
+ */
+const sendCreateContractUpgradeProposalTransaction = async (
+  params: CreateContratctUpgradeProposalParams
+): Promise<TransactionResponse> => {
+  const { title, description, linkToDiscussion, contarctId, proxyId, signer } = params;
+  const upgradeProposalContractId = ContractId.fromString(contarctId).toSolidityAddress();
+  const upgradeProposalProxyId = ContractId.fromString(proxyId).toSolidityAddress();
+
+  console.log(`Roshan data is ${title} ${upgradeProposalContractId} ${upgradeProposalProxyId} ${signer}`);
```

Comment:
Looks like there is a left over console.log here.

Reply:
Done